src/App.tsx

```diff
@@ -34,62 +35,129 @@ export function getReorderedGoods(
   // To avoid the original array mutation
   const visibleGoods = [...goods];
 
-  // Sort and reverse goods if needed
-  // eslint-disable-next-line no-console
-  console.log(sortType, isReversed);
+  if (sortType !== SortType.NONE) {
+    visibleGoods.sort((a, b): number => {
+      switch (sortType) {
+        case SortType.ALPABET:
+          return a.localeCompare(b);
+        case SortType.LENGTH:
+          return a.length - b.length;
+        default:
+          return 0;
+      }
```

Comment:
```suggestion
      switch (sortType) {
        case SortType.ALPABET:
          return a.localeCompare(b);
          
        case SortType.LENGTH:
          return a.length - b.length;
          
        default:
          return 0;
      }
```
add spaces for better readability